content/en/faq/setup/install-method-selection.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Which Istio installation method should I use?
+weight: 10
+---
+
+In addition to the simple [getting started](/docs/setup/getting-started) evaluation install, there are several different
+methods you can use to install Istio. Which one you should use depends on your production requirements.
+The following lists some of the pros and cons of each of the available methods:
+
+1. [istioctl install](/docs/setup/install/istioctl/)
+
+    The simplest and most validated installation/management path with high security.
+    This is generally considered the recommended method, in most cases.
+
+    Pros:
+
+    - Thorough configuration validation and health verification.
+    - Uses `IstioOperator` API which has extensive configuration/customization options.
+    - No in-cluster pods needed. Changes are actuated by running the `istioctl` command.
+
+    Cons:
+
+    - Multiple binaries must be managed, one per Istio minor version.
+    - Some configurations may not be fully declarative and safe to use in CI pipelines (e.g., enabling third-party JWT).
```

Comment:
I worry that this is vague enough to put people off and cause confusion. @nrjpoddar could you reiterate what you consider not declarative? The apply code path is actually identical to the operator, just the actuation is through running `istioctl install`.
Maybe a different way to put the high level concern is that the output manifest is directly applied as part of the command e.g. "If you need to generate and apply the output manifest through some other method (e.g. kubectl), consider using manifest generate or helm."